rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -53,6 +54,9 @@ struct PlayOptions
   // Start paused.
   bool start_paused = false;
 
+  // Time to start playback as an offset from the beginning of the bag.
+  rcutils_time_point_value_t start_offset = INT64_C(0);
```

Comment:
```suggestion
  rcutils_time_point_value_t start_offset = 0;
```
There are no reason for this ridiculous type cast.

Reply:
Noted - my bad! I definitely over-engineered things here. :sweat_smile: 

Addressed in https://github.com/ros2/rosbag2/pull/931/commits/07543e0c770db06fbada46268105fd4eea1a9d3a.